mParticle-Google-Analytics-Firebase/MPKitFirebaseAnalytics.m

```diff
@@ -384,11 +384,11 @@ - (void)logUserAttributes:(NSDictionary<NSString *, id> *)userAttributes {
     if (product.sku) {
         [parameters setObject:product.sku forKey:kFIRParameterItemID];
     }
-    if (product.name) {
-        [parameters setObject:product.name forKey:kFIRParameterItemName];
-    }
-    if (product.category) {
-        [parameters setObject:product.category forKey:kFIRParameterItemCategory];
+    if (product.name && product.category) {
```

Comment:
I think this will need a slight refactor so e.g. name can be set even if category is not present

Reply:
I was debating that. The Firebase docs aren't explicit that they are optional but I bet they are. https://firebase.google.com/docs/reference/ios/firebaseanalytics/api/reference/Constants#/c:FIRParameterNames.h@kFIRParameterItems